fidesctl/src/fidesctl/core/annotate_dataset.py

```diff
@@ -7,7 +7,7 @@
 import click
 
 ## TODO: decouple this from fidesapi, it should be using API calls
-from fidesapi.crud import list_resource
+from fidesapi.routes.crud import list_resource
```

Comment:
I didn't see this also uses `list_resource`... do you think this will also need to have `await list_resource(...` similar to `view.py`?

Reply:
this is removed in the annotate-dataset PR, but good catch!